src/alpaca_eval/models_configs/phi-2/configs.yaml

```diff
@@ -0,0 +1,13 @@
+phi-2:
+  prompt_template: "phi-2/prompt.txt"
+  fn_completions: "vllm_local_completions"
+  completions_kwargs:
+    model_name: "/media/hangyu5/Home/Documents/Hugging-Face/phi-2"
```

Comment:
for merging in the main repo it would be nice to have a more generic folder directory. e.g.     https://github.com/tatsu-lab/alpaca_eval/blob/b7bcc5037cc2a3585f56f94258ead693151afc1c/src/alpaca_eval/models_configs/openbuddy-llama2-70b-v10.1/configs.yaml#L5